plugins/codesample.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Codesample Plugin
-title_nav: Code Sample
+title_nav: Codesample
```

Comment:
I'd prefer that we leave this as two words. There are lots of plugins that have had a similar treatment.